schroot: add session examples, move --list to the top (#10650)
tldr-pages/tldr: pages/linux/schroot.md

> > Run command or start an interactive shell with a different root directory. More customizable than `chroot`.

Reviewer: ```suggestion
> Run a command or start an interactive shell with a different root directory. More customizable than `chroot`.
```

`schroot --chroot {{chroot}} --user {{user}}`

> - Begin a new session. A unique session ID is returned on standard output:

Reviewer 2: ```suggestion
- Begin a new session (A unique session ID is returned on standard output):
```

> - Begin a new session (A unique session ID is returned on standard output):

Reviewer: #10436
```suggestion
- Begin a new session (a unique session ID is returned on `stdout`):
```
